pagopa/io-app
# feat: [IOBP-314] Wallet payment confirm screen

## Short description
This PR introduces the UI to the payment confirmation screen;

## List of changes proposed in this pull request
- Added the UI components into `WalletPaymentConfirmScreen` to show the selected payment method, the PSP with his fees, and the total amount

## How to test
Start the new wallet payment playground and go forward until you reach the end of the flow where you will see a confirmation screen

## Preview
<img src="https://github.com/pagopa/io-app/assets/34343582/85730324-8554-40c4-a756-1e5dcdbd540a" width="300" />
